sofl/installer/online_fix_installer.py

```
@@ -0,0 +1,404 @@
+]
+
+class OnlineFixInstaller:
+    """Class for installing Online-Fix games from RAR archives"""
+    
+    def __init__(self):
+        """Installer initialization"""
+        # Check if installation path exists
+        try:
+            # Try to get the value if the key exists
+            shared.schema.get_string("online-fix-install-path")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)